modules/modal/react/lib/Modal.tsx

```diff
@@ -76,7 +76,11 @@ function onInitialFocus(
   if (firstFocusEl) {
     return firstFocusEl;
   } else {
-    const firstFocusable = modalEl && modalEl.querySelector<HTMLElement>('[aria-label=Close],h3');
+    const firstFocusable =
+      modalEl &&
+      modalEl.querySelector<HTMLElement>(
+        `[data-close=close],#${modalEl.getAttribute('aria-labelledby')}`
```

Comment:
The test is failing with:
> Error: Failed to execute 'querySelector' on 'Element': '[data-close=close],#2f023d86-296a-457c-984f-8d7cbe3650b9' is not a valid selector.

A `uuid` can start with a number, but an id selector shorthand cannot. The following should fix the error:
```suggestion
        `[data-close=close],[id="${modalEl.getAttribute('aria-labelledby')}"]`
```

Reply:
Ah, thank you so much!